crates/pallets/starknet/src/lib.rs

```diff
@@ -492,12 +502,7 @@ pub mod pallet {
 
             // Execute
             let tx_execution_infos = transaction
-                .execute(
-                    &mut BlockifierStateAdapter::<T>::default(),
-                    &Self::get_block_context(),
-                    false,
-                    T::DisableNonceValidation::get(),
-                )
+                .execute(&mut BlockifierStateAdapter::<T>::default(), &Self::get_block_context(), false, false, false)
```

Comment:
`false, false, false` merge those 3 in a struct with explicit field names that is passed as argument